onnxruntime/core/providers/openvino/ov_versions/data_ops.cc

```diff
@@ -118,6 +118,7 @@ std::vector<SupportedOp> supported_op_mode = {
     {"CumSum", V_2022_1, {"CPU", "GPU"}},
     {"DepthToSpace", V_2020_4, {"CPU", "GPU"}},
     {"DequantizeLinear", V_2021_4, {"CPU", "GPU"}},
+    {"DequantizeLinear", V_2024_4, {"CPU", "GPU", "NPU"}},
```

Comment:
Just add NPU here otherwise it wont support DeQuantizeLinear for 2024_3

Reply:
And please merge 2024_5 PR and this one as we do not have support for INT4 in 2024_4

Reply:
> And please merge 2024_5 PR and this one as we do not have support for INT4 in 2024_4

@vthaniel Can you trigger the internal CI to make the PR #480 merge ready

Reply:
> Just add NPU here otherwise it wont support DeQuantizeLinear for 2024_3

NPU is already added like below -
`{"DequantizeLinear", V_2024_4, {"CPU", "GPU", "NPU"}}`

Did you mean this instead for support since 2021.4-
`{"DequantizeLinear", V_2021_4, {"CPU", "GPU","NPU"}},`

Reply:
{"DequantizeLinear", V_2024_3, {"CPU", "GPU", "NPU"}},
    {"DequantizeLinear", V_2024_4, {"CPU", "GPU", "NPU"}}, 

When exactly is this op support started 2024_3 or 2024_4

Reply:
The documentation for this is not known, as per above the support started since 2021.4 for both CPU & GPU (NPU not known).

As per below PR to enable the **DequantizeLinear** support is available since opset version 13

https://github.com/openvinotoolkit/openvino/pull/26915/files


Thus I will retain the same changes in this case as we support all the devices (CPU,GPU,NPU) since 2024.3 and beyond ie. (DequantizeLinear opset 21 in 2024.4 also) to maintain support cadence for end users

Reply:
This statement means the version from where support started .
if you want to add NPU support, just add NPU support from the version u have started testing i.e 2024.5

Reply:
Resolving as per the discussion and amending the suggestions. Tested with new changes successfully.